### ISSUE TITLE
Support for internationalization

### ISSUE THREAD

**Issue by ajmathews** [fknop/angular-pipes#69]
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
```

**Current behavior**
When the `bytes` pipe is used, the unit is displayed in English.

**Expected behavior**
Require a way to translate this into other languages. Does the library already support it?

**Comment by fknop** [comment]
No, it does not support it yet. Maybe you could copy the pipe in the meantime and having a service that setup the units with the correct language would do the trick.
